db/monitor/DELA0F3.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<!-- DELL UltraSharp 38 Monitor (Model 2017) -->
+<monitor name="DELL U3818DW" init="standard">
+	<caps add="(prot(monitor)type(LCD)model(U3818DW)cmds(01 02 03 07 0C E3 F3)vcp(02 04 05 08 10 12 14(04 05 06 08 09 0B 0C) 16 18 1A 52 60(1B 0F 11 12 ) 62 AC AE B2 B6 C6 C8 C9 CC(02 0A 03 04 08 09 0D 06 ) D6(01 04 05) DC(00 03 05 ) DF E0 E1 E2(00 1D 02 04 0C 0D 0F 10 11 13 14) E4(00 01) E5 E7(00 01 02) E8 E9(00 21 22 24 ) F0(0C ) F1 F2 FD)mswhql(1)asset_eep(40)mccs_ver(2.1))"/>
+
+	<!-- Vendor-specific controls for the DELL monitor -->
+	<controls>
+		<control id="PbP" type="list" address="0xe9">
+			<value id="Off"		value="0x00"/>
+			<value id="PbP"		value="0x24"/>
+			<value id="PiP large"	value="0x22"/>
+			<value id="PiP small"	value="0x21"/>
+		</control>
+		<control id="maininputsource" type="list" address="0x60">
```

Comment:
The address `0x60` is standard VESA, for: `<control id="inputsource" ...` . Why did you create another one?